.travis.yml

```diff
@@ -0,0 +1,23 @@
+
+os:
+  - linux
+
+language: c
+
+env:
+  - BRANCH="travis"
+  - BRANCH="devel"
+
+install:
```

Comment:
Probably should be `before_install`; `install` would generally cover things like `nimble install` or `nimble build`